lib/mixlib/install/options.rb

```diff
@@ -35,6 +35,7 @@ class InvalidOptions < ArgumentError; end
         powerpc
         ppc64
         ppc64le
+        rockylinux
```

Comment:
we can remove this entry, as we are only adding a platform and no new architecture.